service/service.py

```diff
@@ -148,6 +148,17 @@ def delete_wishlists(wishlist_id):
         wishlist.delete()
     return make_response('', status.HTTP_204_NO_CONTENT)
 
+######################################################################
+# DELETE ALL PET DATA (for testing only)
+######################################################################
+@app.route('/wishlists/reset', methods=['DELETE'])
```

Comment:
Since we're not using the endpoint, can we remove it?

Reply:
No, I want to still use it. I can cancel the PR if that is better since tomorrow I will work on it.

Reply:
Not needed. We can let it be open. You can put a title saying do not merge or work in progress or something